src/Components/Components/test/Forms/EditContextTest.cs

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Forms;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.Tests.Forms
```

Comment:
nit: remove `.Tests` from these namespaces and change the `BaseNamespace` of the project to match the product-side. This makes it trivial to write unit tests.

Reply:
Done